plugins/obs-outputs/ftl-stream.c

```diff
@@ -109,8 +109,6 @@ struct ftl_stream {
 	frame_of_nalus_t coded_pic_buffer;
 };
 
-static void log_libftl_messages(ftl_log_severity_t log_level,
-				const char *message);
```

Comment:
This function was added by 6f8e38e0a08f2a7a1ad422b83516be106f771748 but was never used at that time.